ctapipe/io/magiceventsource.py

```diff
@@ -0,0 +1,1337 @@
+
+
+import glob
+import re
+
+import uproot
```

Comment:
With this, uproot is now a required dependency of ctapipe.

So it has to be added to the setup.py and environment.yaml.
Or you make it optional by wrapping it in something like `try... except ImportError`